.travis.yml

```diff
@@ -1,5 +1,16 @@
 language: scala
 
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt
+
+before_cache:
+  # Cleanup the cached directories to avoid unnecessary cache updates
```

Comment:
Hmm maybe just delete all the contents of `~/.ivy2/cache` and `~/.sbt` ... why just specifically these files? :thinking:

Reply:
If not, then include the reason for only removing `*.properties`/`*.lock` in the comment

Reply:
It seems that it doesn't really matter. 
The original comes from here: https://www.scala-sbt.org/0.13/docs/Travis-CI-with-sbt.html#Caching
There is no explanation though. 

Here is the way you suggested: https://github.com/typelevel/cats/pull/2158
and I quote: `seems marginally faster, or at least not slower, and it is also simpler.`

So for the sake of simplicity lets remove entire folders. :D